src/ui/templates/search/autocomplete.hbs

```diff
@@ -1,4 +1,21 @@
 <div class="yxt-AutoComplete-wrapper js-yxt-AutoComplete-wrapper" aria-expanded="{{hasResults}}">
+  <span class="yxt-AutoComplete-resultsCount sr-only" aria-live="assertive">
+    {{#unless beforeSearch}}
+      {{#if hasResults}}
+        {{#each sections}}
+          {{translate
+            phrase='[[resultsCount]] [[label]] autocomplete option found.'
```

Comment:
Outside the context of filter search, there is no `label` attribute. I think out of an abundance of caution, we should have a phrase for when `label` is present and another phrase for when there is no `label`. Having a single phrase like this, we may run into issues for certain languages when `label` is the empty string or undefined.